libraries/Microsoft.Bot.Builder.Dialogs.Adaptive.Testing/Schemas/Microsoft.Test.HttpRequestSequenceMock.schema

```diff
@@ -32,6 +32,14 @@
                 "https://contoso.com"
             ]
         },
+        "content": {
+            "type": "string",
+            "title": "Content",
+            "description": "The content to match partially against.",
+            "examples": [
+                "content"
+            ]
+        },
         "responses": {
```

Comment:
I'm a little confused on how this is used.  Is this for matching against things like the post body?  If so, body would probably be a better name.

Reply:
Add a new field BodyMatchType: Exact and Partial. I just use methods from RichardSzalay.MockHttp now. In the future, we could extend by adding regex etc.